code/game/objects/structures/vehicles/wheelchair.dm

```diff
@@ -152,3 +152,100 @@
 	..()
 
 	update_mob() //Update the rest
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized
+	name = "motorized wheelchair"
+	nick = "cripplin' revenge"
+	desc = "A chair with fitted wheels which is powered by an internal cell. It propels itself without the need for hands as long as it is charged."
+	var/maintenance = 0
+	var/default_cell_path = /obj/item/weapon/cell/high
+	var/obj/item/weapon/cell/internal_battery = null
+
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/New()
+	..()
+	internal_battery = new default_cell_path(src)
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/examine(mob/user)
+	..()
+	if(internal_battery)
+		user << "<span class='info'>The battery meter reads: [round(internal_battery.percent(),1)]%</span>"
+	else
+		user << "<span class='warning'>The 'check battery' light is blinking.</span>"
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/Move()
+	..()
+	internal_battery.use(2) //Example use: 100 charge to get from the cargo desk to medbay side entrance
```

Comment:
Needs sanity for when the battery is removed. Currently not an issue during testing because it's supposed to exit right after anyway, and the use(2) would be irrelevant with the battery out. I'll fix this in the final round of commit changes, after chemfun hits.